src/app/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -38,12 +38,17 @@ const Breadcrumbs = ({ query = '', type, bibUrl, itemUrl, edd }) => {
       return crumbs;
     }
 
-    crumbs.push(
-      <li key="search">
-        <Link to={`${baseUrl}/search?${query}`} onClick={() => onClick('Search Results')}>
-          Search Results
-        </Link>
-      </li>);
+    const stringifiedQuery = query.replace(/^q=/, "");
+
+    if (stringifiedQuery && stringifiedQuery !== "undefined") {
```

Comment:
Your breadcrumbs work is slipping into this PR.

Reply:
Darn :( sorry